src/main/proto/ga4gh/variants.proto

```diff
@@ -191,4 +191,29 @@ message Variant {
   // determination of genotype with respect to this variant. `Call`s in this
   // array are implicitly associated with this `Variant`.
   repeated Call calls = 12;
+
+  // The "variant_type" is used to denote e.g. structural variants.
+  // Examples:
+  //   DUP  : duplication of sequence following "start"; not necessarily in situ
+  //   DEL  : deletion of sequence following "start"
+  string variant_type = 13;
```

Comment:
@david4096 But not all of the variant types may (well, will) be supported by the schema (yet). So:

* you need either A) a positive (i.e. defining a set of) or B) a selective (i.e. which of all defined in external reference) variants would constitute supported values
* case A would naturally be an enumeration, B a documentation case with ontologyTerm (+ service)
Both have conceptual pros and cons, with in A ease/stringency, in B beauty/future speaking for them.

???